web-services/query/src/main/java/datawave/webservice/query/runner/QueryExecutorBean.java

```diff
@@ -2997,6 +2997,26 @@ private void updateQueryParams(Query q, String queryLogicName, String query, Dat
         }
     }
 
+    /**
+     * @param queryLogicName
+     *            the logic name
+     * @param queryParameters
+     *            the query parameters
+     * @return the generic response
+     */
+    @POST
+    @Produces({"application/xml", "text/xml", "application/json", "text/yaml", "text/x-yaml", "application/x-yaml", "application/x-protobuf",
+            "application/x-protostuff"})
+    @Path("/{logicName}/validateQuery")
```

Comment:
Lets just use /{logicName}/validate to be consistent with the create, next, close calls

Reply:
The function name is fine however.

Reply:
Fixed.